### PR TITLE
007_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName=config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = value_steering_wheel_location })
+	EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
+end
+
+function Test:TestStep_RAI_SecondApp_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession1:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.registerAppInterfaceParams.appName }})
+	self.mobileSession1:ExpectResponse(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = value_steering_wheel_location })
```
Comment (aderiabin) [inline test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua]: As I understood parameter `steeringWheelLocation` should be part of `hmiCapabilities`.
Could you, please, check?

File: test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		self.mobileSession2:StartService(7)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_RAI_FirstApp_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName=config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success=true, resultCode = "SUCCESS", steeringWheelLocation = value_steering_wheel_location })
```
Comment (aderiabin) [inline test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua]: As I understood parameter steeringWheelLocation should be part of hmiCapabilities.
Could you, please, check?

Comment (istoimenova) [inline test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua]: @aderiabin, done in 4644e8e90d09067cadd84714fa32c190195a52cf

File: test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/RAI/035_ATF_N_HMI_Does_Not_Respond_3_Apps_RAI_steeringWheelLocation.lua]: @aderiabin, done in 4644e8e90d09067cadd84714fa32c190195a52cf

**Pull request merged**